tests/unit/labels/MockLabels.h

```diff
@@ -12,6 +12,14 @@ namespace shogun {
 			MOCK_METHOD0(get_values, SGVector<float64_t>());
 
 			virtual const char* get_name() const { return "MockCLabels"; }
+
+		protected:
+			virtual CSGObject *create_empty() const
```

Comment:
i dont understand why you need this as there is never an empty instance created ... or am I wrong?

Reply:
There were situations in which we were emitting `MockLabels` from a machine and therefore I needed to implement the `create_empty()` method to make `clone()` work.